composer.json

```diff
@@ -49,6 +49,8 @@
             "php": "5.5.9"
         }
     },
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```

Comment:
Isn't this really dangerous? I mean, isn't there a way to release a proper release for the APCu polyfill, even a beta/rc for the minimum stability to rise up and not be `dev`?